src/sentry/api/serializers/models/project.py

```diff
@@ -391,26 +392,26 @@ def measure_span(op_tag):
                     serialized["options"] = options[project.id]
         return result
 
-    def get_stats(self, project_ids, query):
+    def get_stats(self, projects, query):
```

Comment:
Actual changes here, changed the param to a list of param objects so we don't need to hit the db again